src/legacy/core_plugins/tests_bundle/index.js

```diff
@@ -104,11 +104,14 @@ export default (kibana) => {
           modules: [...modules],
           template: createTestEntryTemplate(uiSettingDefaults),
           extendConfig(webpackConfig) {
-            return webpackMerge({
+            const mergedConfig = webpackMerge({
               resolve: {
                 extensions: ['.karma_mock.js', '.karma_mock.tsx', '.karma_mock.ts']
               }
             }, webpackConfig);
+            delete mergedConfig.optimization.splitChunks.cacheGroups.commons;
```

Comment:
what's up with this line?  How come it fixed the browser tests?

Reply:
@spalger could you please comment on this?

Reply:
Actually I have Spencer's explanation here:

> [..] it removes the commons bundle creation from the webpack config when we're building the bundle for the browser tests. It shouldn't be created, and by default isn't, but something is triggering it in webpack which breaks the tests so if we just remove the optimization config it will never happen and the tests will keep working [..]

Reply:
maybe add it as a comment?  Otherwise I could see a developer not understanding why it is in there.

Reply:
Added comment.